src/utils/apis/OfferApi.js

```diff
@@ -74,7 +74,31 @@ export const offersAPI = {
         })
     },
 
+    update: function (offerData) {
+        const offerId = offerData.offerId
+        delete offerData.offerId
+        return fetch(recruitmentServiceBasicAPILink + `/api/offers/update/${offerId}`, {
+            method: "PUT",
+            headers: {
```

Comment:
Tak jak mieliśmy w IO @Rados13 - headery można chyba wynieść do jakiejś zmiennej - zwłaszcza jeśli potem pojawią się jeszcze tokeny.

Reply:
No ja jestem ogólnie za tym by jak można wyciągnąć kod powtarzający to fajnie to zrobić XD